python/plugins/processing/algs/gdal/translate.py

```diff
@@ -73,6 +82,18 @@ def initAlgorithm(self, config=None):
                                                         self.tr('Copy all subdatasets of this file to individual output files'),
                                                         defaultValue=False))
 
+        self.addParameter(QgsProcessingParameterString(self.OUTSIZE,
+                                                       self.tr('X and Y size of output. Pixel size unless ‘%’ is appended, then will be a fraction of original size. One 0 value will be estimated to preserve aspect ratio.'),
+                                                       defaultValue=self.tr('100% 100%'))
```

Comment:
This one shouldn't be translated - it's a literal value for gdal

Reply:
should be done